components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/util/Utils.java

```diff
@@ -719,6 +719,24 @@ public static String getCallbackURL(org.wso2.carbon.identity.recovery.model.Prop
         return callbackURL;
     }
 
+    /**
+     * Get isAccessUrlAvailable property value.
+     *
+     * @param properties Properties array.
+     * @return Boolean value of the isAccessUrlAvailable property.
+     */
+    public static Boolean isAccessUrlAvailable(org.wso2.carbon.identity.recovery.model.Property[] properties) {
```

Comment:
Change the Boolean to boolean here

Reply:
Fixed with https://github.com/wso2-extensions/identity-governance/pull/815